src/ragas/metrics/critique.py

```diff
@@ -81,6 +83,14 @@ def __post_init__(self: t.Self):
         self.strictness = (
             self.strictness if self.strictness % 2 != 0 else self.strictness + 1
         )
+        self.critic_prompt = CRITIQUE_PROMPT
+
+    def adapt(self, language: str, cache_dir: str | None = None) -> None:
+        logging.info(f"Adapting Critic to {language}")
```

Comment:
create a new logger from logging like this
this goes on top of the file after all imports. The inits a new logger for the module. this means users can set logging level for ragas separately and see where the logs came from
```py
# logging_example.py

import logging

# Create a custom logger
logger = logging.getLogger(__name__)
```

then use
```py
logger.info(f"Adapting Critic to {language}")
```

Reply:
do this for everything